src/styles/block/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  @include grid-colums-count;
+
+  display: grid;
+  grid-template-columns: repeat($grid-colums-1, $card-width);
```

Comment:
The variable '$card-width' should be used to set the width of each column, but the 'repeat' function should use the number of columns as its first argument. Please check and ensure that the first argument to 'repeat' specifies the correct number of columns, and that '$card-width' is used to define the width of each column.